wordpress-mobile/WordPress-iOS
# Enable new About screen for Jetpack

This PR enables the new About screen for Jetpack. I've updated the screen to use the existing `AppConstants`, and added a few extras so that they can share an About Screen configuration.

<img src="https://user-images.githubusercontent.com/4780/143613245-d041ba9b-1520-4e08-81c0-841ab59250e1.png" width=350>

I'm not sure if the `currentAppIconName` code really belongs in the AppConstants file or not, I could see a case for it being moved elsewhere. Let me know if you think it should go somewhere else!

**To test**

* Build and run Jetpack
* Navigate to Me > About Jetpack for iOS
* Check the screen looks good and that each item navigates where you'd expect. Rate Us, Share, Twitter, and Blog should all take you to Jetpack-specific content.
* Also Build and run WordPress and double check the About screen there still reflects WordPress and not Jetpack.

## Regression Notes

1. Potential unintended areas of impact

None really, unless the WordPress about screen accidentally now referenced Jetpack somewhere.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

I retested each item in the WordPress About screen.

3. What automated tests I added (or what prevented me from doing so)

None

PR submission checklist:
